src/System.Private.CoreLib/src/System/Runtime/InteropServices/SafeHandle.CoreCLR.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    public abstract partial class SafeHandle
+    {
+        private void FinalizeCore() => Dispose(disposing: false);
```

Comment:
I would rather ifdef out the finalizer for CoreRT instead of splitting this into two methods.

Reply:
CoreRT does this odd thing where its ~SafeHandle() does `new DelayedFinalizer(this)`, DelayedFinalizer has a finalizer that then invokes Dispose(false) on the SafeHandle.  So, just to make sure I'm understanding correctly, are you saying you want to move that into an ifdef in the shared SafeHandle.cs file?  Or ifdef out the finalizer in SafeHandle.cs entirely for corert?  Or something else?

Reply:
(The CoreRT impl of the finalizer is a hack to workaround another hack. It is not a proper durable fix.)

Reply:
Ok.  So just ifdef it out entirely. That's easy ;)

Reply:
Right, the CoreRT impl is a workaround for `CriticalFinalizerObject` not being implemented properly.